pyds9/pyds9.py

```diff
@@ -116,75 +118,92 @@ def get_xpans_ds9():
 
 # load pyfits, if available
 try:
-    from astropy.io import fits as pyfits
-    ds9Globals["pyfits"] = 1
-except:
-    try:
-        import pyfits
-        if pyfits.__version__ >= '2.2':
-            ds9Globals["pyfits"] = 2
-        else:
-            ds9Globals["pyfits"] = 0
-    except:
-        ds9Globals["pyfits"] = 0
+    import pyfits
+    if pyfits.__version__ >= '2.2':
+        ds9Globals["pyfits"] = True
+    else:
+        warnings.warn('Pytest<2.2 is not supported')
+        ds9Globals["pyfits"] = False
+except ImportError:
+    ds9Globals["pyfits"] = False
 
-# load numpy, if available
-try:
-    import numpy
-    ds9Globals["numpy"] = 1
-except:
-    ds9Globals["numpy"] = 0
 
 # numpy-dependent routines
-if ds9Globals["numpy"]:
-    def _bp2np(bitpix):
-        """
-        Convert FITS bitpix to numpy datatype
-        """
-        if bitpix == 8:
-            return numpy.uint8
-        elif bitpix == 16:
-            return numpy.int16
-        elif bitpix == 32:
-            return numpy.int32
-        elif bitpix == 64:
-            return numpy.int64
-        elif bitpix == -32:
-            return numpy.float32
-        elif bitpix == -64:
-            return numpy.float64
-        elif bitpix == -16:
-            return numpy.uint16
-        else:
-            raise ValueError('unsupported bitpix: %d' % bitpix)
+def _bp2np(bitpix):
```

Comment:
Since numpy is always present, there is no need to guards ``_bp2np``and ``_np2bp``.